applications/mne_analyze/mne_analyze/mainwindow.cpp

```diff
@@ -85,8 +85,9 @@ MainWindow::MainWindow(QSharedPointer<ANSHAREDLIB::PluginManager> pPluginManager
 , m_sSettingsPath("MNEANALYZE/MainWindow")
 , m_sCurrentStyle("default")
 {
+    this->setObjectName("mainwindow");
     setWindowState(Qt::WindowMaximized);
-    setMinimumSize(400, 400);
+    setMinimumSize(1280, 720);
```

Comment:
Maybe use 800x450? just in case someone with a very small (and old) laptop uses MNE Analyze ;)